go/obscuronode/enclave/db.go

```diff
@@ -314,5 +314,8 @@ func (db *inMemoryDB) ParentBlock(block *types.Block) (*types.Block, bool) {
 	db.blockM.RLock()
 	defer db.blockM.RUnlock()
 	b, f := db.blockCache[block.ParentHash()]
+	if !f {
+		return nil, false
```

Comment:
@tudor-malene Would it be better to just return a single value from this method? Instead of returning `false` to indicate the block wasn't found, you would check whether the returned block is `nil`.

Reply:
I just followed the pattern used in golang.
Will think about it a bit